README.md

```diff
@@ -77,4 +89,4 @@ This repository contains helper scripts and commands to create meetup videos for
 
 * create the uploadable version
 
-        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" uploadable.mp4
+        $ ffmpeg -i title.mp4 -i final2.mp4 -filter_complex "[0:0] [0:1] [1:0] [1:1] concat=n=2:v=1:a=1 [v] [a]" -map "[v]" -map "[a]" -preset ultrafast uploadable.mp4
```

Comment:
issue wasn't about long time, without -preset ultrafast it was using all four cores to 100 %, if load sharing could be used that would useful somehow. 
with ultrafast the cpu cores were around 60-65% .